.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+ - [ ] Defect
+ - [ ] Feature Request or Change Proposal
+ 
+## Defects 
+
+Make sure that these boxes are checked before submitting your issue -- thank you!
+
+ - [ ] Included `gnatsd -DV` output
+ - [ ] Included a [Minimal, Complete, and Verifiable example] (http://stackoverflow.com/help/mcve)
+ 
+ #### Versions of `gnatsd` and affected client libraries used:
+
+ #### OS/Container environment:
+
+ #### Steps or code to reproduce the issue:
+ 
+ #### Expected result:
+ 
+ #### Actual result:
+ 
+ 
+ ## Feature Requests
```

Comment:
You should remove the first space, otherwise, it looks like "Feature Requests" is aligned with "Actual result:" instead of "Defects".